source/reference/minio-server/settings/metrics-and-logging.rst

```diff
@@ -0,0 +1,865 @@
+.. _minio-server-envvar-metrics-logging:
+
+======================================================
+Environment Variables to Configure Metrics and Logging
+======================================================
```

Comment:
We need to work on the page structure here.

The H1s don't 1:1 nicely - `Environment Variables -> Server Logs` when the config analog is `Configuration Values - HTTP Webhook Log Target`.

This is another area where I'd recommend swapping the H1s and H2s:

```
# Prometheus Authentication

# Server Webhook Logging

## Envvar


## Config Setting

# Audit Webhook Logging

## Envvar

## Config Setting

```